lib/stat_tracker.rb

```diff
@@ -57,6 +57,10 @@ def count_of_games_by_season
     @games.each_with_object(Hash.new(0)) { |game, game_count| game_count[game.season] += 1 }
```

Comment:
Looks good. Though refactoring is nice, we could also keep it out longer for for readability. But since I'll be using this in my method I'm building, could be nice to have it just be short form.